pesa_migration/pesa_add_new_lang_symbols.sql

```diff
@@ -42,3 +42,9 @@ select max(lang_id)+1 ,'y', -1, 'lang', 'year_of_birth', 'year of birth', 'Gebur
 
 insert into ##new_db##.or_lang
 select max(lang_id)+1 ,'y', -1, 'lang', 'you_have_to_year_of_birth', 'Please enter your year of birth.', 'Bitte geben Sie Ihr Geburtsjahr an.' from ##new_db##.or_lang;
+
+insert into ##new_db##.or_lang
+select max(lang_id)+1 ,'y', -1, 'lang', 'cancellation_end_hours_before', 'Cancellation deadline (hours before start)', 'Abmeldungsende (Stunden vor Experimentbeginn)' from ##new_db##.or_lang;
```

Comment:
Here maybe use "Sessionbeginn" instead "Experimentbeginn"?  Otherwise it can be confusing...